test/unit/java.test.ts

```diff
@@ -20,6 +20,7 @@
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import fs from 'fs';
+import fsExtra from 'fs-extra';
```

Comment:
we should the `fs` import and just use `fs-extra`

Reply:
good catch 👍🏻